test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -345,7 +345,7 @@ public void CreatePackageWithNuspecIncludeExcludeWithWildcards(string source, st
                 builder.AddFiles(directory.Path, source, destination, exclude);
 
                 // Assert
-                Assert.Collection(builder.Files,
+                Assert.Collection(builder.Files.OrderBy(f => f.Path.Length),
```

Comment:
Can we use a more reliable sort here, rather than by length, in case there's multiple things with the same length?

Reply:
Sure, will do tomorrow. Want to see test results :)

Reply:
Done